apps/dotcms-ui/src/app/api/services/dot-paginator-es-content/dot-paginator-es-content.service.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { TestBed, getTestBed } from '@angular/core/testing';
+import { HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { CoreWebService } from '@dotcms/dotcms-js';
+import { CoreWebServiceMock } from '@tests/core-web.service.mock';
+import { DotPaginatorESContentService } from './dot-paginator-es-content.service';
+
+describe('DotPaginatorESContentService', () => {
+    let injector: TestBed;
+    let dotPaginatorESContentService: DotPaginatorESContentService;
+    let httpMock: HttpTestingController;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [HttpClientTestingModule],
+            providers: [
+                { provide: CoreWebService, useClass: CoreWebServiceMock },
+                DotPaginatorESContentService
+            ]
+        });
+        injector = getTestBed();
+        dotPaginatorESContentService = injector.get(DotPaginatorESContentService);
+        httpMock = injector.inject(HttpTestingController);
+    });
+
+    it('should get Blogs with default values', () => {
+        dotPaginatorESContentService.get({ query: '+contentType: blog' }).subscribe((res) => {
```

Comment:
What else besides content type can we have in the query property? because if it's only content types is better to have a content type property in the interface.

Reply:
any attribute related to that contentlet like working, live, deleted, etc
![image](https://user-images.githubusercontent.com/37185433/143117852-5f89d650-4b17-485e-b988-35c1775c9214.png)